beastling/configuration.py

```diff
@@ -294,6 +188,8 @@ def process(self):
         BeastXml object can be instantiated from this Configuration with no
         problems.
         """
+        if self.processed:
+            return
```

Comment:
Since the clocks and models now keep a reference to the configs they have been initialized with I could just store configs and after processing the initialized objects in the same attribute (`clocks` and `models` respectively). But this means that one can no longer run the code in `process` multiple times.

I could revert this, but I think it is more transparent if there are just the two states "processed" and "not processed". Or are there any use cases where one would hold on to one `Configuration` object, fiddle around with the state and run `process` multiple times?